defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
 gluster_cluster_volume_base_path: /data/glusterfs
-gluster_cluster_storage_group_name: storage
+gluster_cluster_storage_group_name: cluster_storage
 gluster_cluster_hosts: "{% for item in groups[gluster_cluster_storage_group_name] %}{{ hostvars[item]['ansible_' + gluster_cluster_transport_interface]['ipv4']['address'] }}{% if not loop.last %},{% endif %}{% endfor %}"
+gluster_converged_disks: false
```

Comment:
Is this the lingo used by Gluster? I couldn't see anything referencing it. Does it just mean that software RAID will be used on the bricks?